### PR TITLE
V3: Adds version and build number to the hello codap area

### PR DESCRIPTION
**Pull request by eireland**

### CODE REVIEW

File: v3/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "codap3",
-  "version": "0.0.1",
+  "version": "v3.0.0-build.700",
```
Comment (kswenson) [inline v3/package.json:3]: Version listed here should not have the "V" prefix. It's fine to display it that way to the user if desired, but this field is processed by npm and others. I gather the "700" was an attempt to push it beyond the current v2 build number? If that's a goal I'd probably start at "1000" just to make sure they don't overlap, but I'm not sure that actually matters. I was imagining we'd just start low and build.

**Review by kswenson — CHANGES_REQUESTED** [review]
👍 This captures the basic idea. Need to figure out what we actually want to do with the build number.

**Review by bfinzer — APPROVED** [review]
👍🏻 LGTM
To what extent can we automate updating the version information? And when should it be automated? Perhaps the build number gets incremented every time there's a merge to main?

**Comment by kswenson** [comment]
I talked to @bfinzer and he suggested starting the build numbers at `1000`.

**Comment by kswenson** [comment]
> 👍🏻 LGTM To what extent can we automate updating the version information? And when should it be automated? Perhaps the build number gets incremented every time there's a merge to main?

Automating the build number incrementing will require some research and probably shouldn't hold up this PR.

**Pull request merged**